File: src/model/Item.java
package model;/*
author :Himal
version : 0.0.1
*/

public class Item {
    private String itemCode;
    private String itemName;
    private double itemUnitPrice;
    private int itemStock;

    public Item(String itemCode, String itemName, double itemUnitPrice, int itemStock) {
        this.itemCode = itemCode;
        this.itemName = itemName;
        this.itemUnitPrice = itemUnitPrice;
        this.itemStock = itemStock;
    }

    public Item() {
    }

    public String getItemCode() {
        return itemCode;
    }

    public void setItemCode(String itemCode) {
        this.itemCode = itemCode;
    }

    public String getItemName() {
        return itemName;
    }

    public void setItemName(String itemName) {
        this.itemName = itemName;
    }

    public double getItemUnitPrice() {
        return itemUnitPrice;
    }

    public void setItemUnitPrice(double itemUnitPrice) {
        this.itemUnitPrice = itemUnitPrice;
    }

    public int getItemStock() {
        return itemStock;
    }

    public void setItemStock(int itemStock) {
        this.itemStock = itemStock;
    }
}

File: web/model/item.js
function Item() {
    var __itemCode;
    var __itemName;
    var __itemUnitPrice;
    var __itemStock;

    this.setItemCode = function (itemCode) {
        __itemCode = itemCode;
    }

    this.getItemCode =function () {
        return __itemCode;
    }

    this.setItemName = function (itemName) {
       __itemName = itemName;
    }

    this.getItemName = function () {
        return __itemName;
    }

    this.setItemUnitPrice =function (itemUnitPrice) {
         __itemUnitPrice = itemUnitPrice;
    }
    this.getItemUnitPrice = function () {
            return __itemUnitPrice
    }

    this.setItemStock = function (itemStock) {
        __itemStock = itemStock;
    }

    this.getItemStock = function () {
        return __itemStock;
    }
}